### DISCUSSION TITLE
Agentic RAG with Langchain: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#70] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/langchain
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=MP-Ke-NBaKo
Slides: [Slides](https://www.canva.com/design/DAGPCMini0M/0V3Z_IJ0A9wpJn5N0xZDTw/edit?utm_content=DAGPCMini0M&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)
Code: https://github.com/Azure-Samples/contoso-creative-writer

Please ask any follow-up questions here, and the speakers will see them.

**Comment by jonnyforsterMSG** [comment]
Hi, maybe I'm being blind but I can't see a langchain branch?
